db/fixtures/miq_product_features.yml

```diff
@@ -6088,6 +6092,10 @@
         :description: Refresh Block Storage Managers
         :feature_type: control
         :identifier: ems_block_storage_refresh
+      - :name: Re-check Authentication Status
+        :description: Re-check Authentication Status of storage manager
+        :feature_type: control
+        :identifier: ems_storage_recheck_auth_status
```

Comment:
Interesting, I wonder if this should be `ems_block_storage_recheck_auth_status` in the code.  It's fine but the inconsistency is weird.

Note, the [ems_storage](https://github.com/ManageIQ/manageiq/blob/ec01a5afe0a8286bff871b249992fc5ed894fbfa/db/fixtures/miq_product_features.yml#L5990-L6049) section is right above this.   Very confusing.

Reply:
In ui-classic, it is in block storage toolbar, but anyways, i am in process of combining block and object storage to one for another enhancement, then @agrare going to refactor this file.

Reply:
Ok, just keep in mind if identifier gets renamed, we need to do a migration 😢